### PR TITLE
A few things seem to have changed in the latest MDX library

### PR DESCRIPTION
**Pull request by alexsanjoseph**
Source: https://github.com/kentcdodds/mdx-bundler/pull/152

### CODE REVIEW

File: posts/create-a-markdown-blog/create-a-markdown-blog.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ export async function getStaticProps({ params }) {
   // post path
   const currentDirectory = process.cwd()
   const postPath = `${currentDirectory}/posts/${slug}/${slug}.mdx`
-  const markdown = await bundleMDXFile(postPath)
+  const markdown = await bundleMDX({file: postPath, ...options})
```
Comment (mattcroat) [inline posts/create-a-markdown-blog/create-a-markdown-blog.md:355]: Please fix the formatting. 😄

**Comment by mattcroat** [comment]
Thank you!

Can you please format the `bundleMDX` lines so the spacing is consistent?

```diff
- bundleMDX({file: postPath })
+ bundleMDX({ file: postPath }) 
```

**Comment by alexsanjoseph** [comment]
Of course - I hope these are all the changes? `npm run format` threw up a lot more changes for me, so I'm not committing them

**Comment by mattcroat** [comment]
> Of course - I hope these are all the changes? `npm run format` threw up a lot more changes for me, so I'm not committing them

Thank you! 😄

**Pull request merged**